### PR TITLE
LPS-117341

### PR DESCRIPTION
**Pull request by RitaSilveira**
JIRA Ticket:
LPS-117341
LPS-135297

Motivation:
The label "..." indicating that more categories are needed to avoid misalignment of lines in the content dashboard

### CODE REVIEW

**Comment by RitaSilveira** [comment]
https://issues.liferay.com/browse/LPS-134022, cc: @CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -2016,4 +2016,55 @@ definition {
...
+		}
+
+		task ("Nagigate to Content Dashboard and check if there's a label with '...' indicating 'more categories' "){
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
+
+			Pause(locator1 = "5000");
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: We don't use Pause anymore, please use Wait methods instead

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 2]
```diff
@@ -2016,4 +2016,55 @@ definition {
...
+	    property portal.acceptance = "true";
+
+     	Category.openCategoriesAdmin(siteURLKey = "global");
+
+        task ("Create a list of Audiences"){
+			for (var count : list "1,2,3"){
+				ContentDashboard.addAudience(
+					title  = "teste ${count}");
+			}		
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase]: Why do you create a new method when you can use JSON macro to create categories?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 3]
```diff
@@ -231,6 +231,50 @@ definition {
...
+			WebContentNavigator.gotoEditCP(webContentTitle = "WC");
+
+			for (var count : list "1,2,3") {
+				WebContent.selectCategory(
+					categoryName = "teste ${count}",
+					vocabularyName = "Audience");
+			}
+
+			PortletEntry.publish();
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:264]: This is not needed, you can use JSON macro to create a web content with several categories.

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
@RitaSilveira SF failed :(

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:forward

**Pull request closed without merging**